tests/framework/validators/StringValidatorTest.php

```
@@ -157,4 +225,52 @@ public function testValidateValueInNonStrictMode(): void
+{
+    public $attr;
+
+    public function attributeLabels()
+    {
+        return ['attr' => 'Test Attribute'];
+    }
+}
+
+class StringViewStub extends View
+{
+    public function registerAssetBundle($name, $position = null)
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$name'. (undefined)

(UnusedFormalParameter)